pcweb/pages/docs/advanced_guide/memoization.py

```diff
@@ -0,0 +1,50 @@
+import pynecone as pc
+
+from pcweb.templates.docpage import doccode, docdemo, docheader, docpage, doctext
+
+
+code1 = """
+@pc.memo
+def fixed_header(text: str) -> pc.Component:
+    \"""A fixed header that doesn't change when the state changes.\"""
+    return pc.heading(text)
+"""
+exec(code1)
+code2 = """fixed_header(text="This Heading Won't Rerender")"""
+
+
+@docpage()
+def memoization():
+    return pc.box(
+        docheader("Memoization", first=True),
+        doctext(
+            "By default, Pynecone will re-render the entire page every time the state changes. "
+            "This is fine for small apps, but can be slow for larger apps. "
+        ),
+        doctext(
+            "You can make rendering faster by memoizing components. ",
```

Comment:
reword to something like:
You can prevent unnecessary rendering  by memoizing components, which can greatly improve your apps performance.

Reply:
updated this